app/javascript/packs/pages/binx_admin.js

```diff
@@ -65,4 +68,29 @@ window.BinxAdmin = class BinxAdmin {
       return image_btn.toggleClass("using_bikes");
     });
   }
+  adminLocations() {
+    $('form').on('click', '.remove_fields', function(event) {
+      // We don't need to do anything except slide the input up, because the label is on it.
+      return $(this).closest('fieldset').slideUp();
+    });
+      // event.preventDefault()
+    return $('form').on('click', '.add_fields', function(event) {
+      const time = new Date().getTime();
+      const regexp = new RegExp($(this).data('id'), 'g');
+      $(this).before($(this).data('fields').replace(regexp, time));
+      event.preventDefault();
+      const us_val = parseInt($('#us-country-code').text(), 10);
+      for (let location of Array.from($(this).closest('fieldset').find('.country_select_container select'))) {
+        const l = $(location);
+        if (!(l.val().length > 0)) { l.val(us_val); }
+      }
+      const names = $(this).closest('fieldset').find('.location-name-field input');
+      for (let name of Array.from(names)) {
+        const n = $(name);
+        if (!(n.val().length > 0)) { n.val($('#organization_name').val()); }
+      }
+
+      return $('.chosen-select select').select2();
```

Comment:
This throws an error because select2 isn't defined.

I'm opening another PR to add fancy select classes, but this isn't finished because this doesn't work

Reply:
Fancy selects were (just) added in #690 - I have already merged it in to this branch.

You don't need to have this return anything. All you need to do is replace this line with `LoadFancySelects()`

You'll also need to add the classes `.unfancy.fancy-select` to the form group (see [an example here](https://github.com/bikeindex/bike_index/pull/690/files#diff-acfdc28655d470fe581af313b3141c77R9)) for each of the selects that should be fancy (hint - any ones that used to have the class `.chosen-select`) - this goes for all future selects as well.